code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -82,9 +82,9 @@ var/global/list/obj/item/device/pda/PDAs = list()
 	if(..(user, 1))
 		to_chat(user, "The time [worldtime2text()] is displayed in the corner of the screen.")
 		if(id)
-			to_chat(user, "There is \a [id] in the ID slot.")
+			to_chat(user, "There is \a [id] in the ID slot. Alt-click to remove \the [id].")
 		if(inserted_item)
-			to_chat(user, "There is \a [inserted_item] in the pen slot.")
+			to_chat(user, "There is \a [inserted_item] in the pen slot. Ctrl-click to remove \the [inserted_item].")
```

Comment:
Not really a fan of giving usage descriptions via examine.

Reply:
moved to desc_fluff